### ISSUE TITLE
Instant and Quick Sell no longer showing

### ISSUE THREAD

**Issue by Ryahn** [jshackles/Enhanced_Steam#1387]
I am pretty sure I saw these buttons at some point. I dont use anything else other than ES when visiting steam.

The only thing I get right now, is just a sell button or sometimes I wont get a sell button.

**Comment by seeeeew** [comment]
Which browser are you using? I think the instant and quick sell buttons are currently disabled in Firefox due to a bug.
http://steamcommunity.com/groups/enhancedsteam#announcements/detail/486794522062750427

**Comment by Ryahn** [comment]
@seeeeew Thanks for the reply. I am actually using chrome on windows 10

**Comment by seeeeew** [comment]
Is "Community → Inventory → Show Quick Sale buttons on Steam Community items in inventory" in the Enhanced Steam options activated?

**Comment by Ryahn** [comment]
Just doubled checked, and they are
![image](http://i.imgur.com/MX2iud4.png)

**Comment by JoiNNN** [comment]
Are you using multiple accounts?

**Comment by jshackles** [comment]
This appears to be an issue with Steam's API, I am investigating.

**Comment by JoiNNN** [comment]
Does the issue still manifest in 9.5?

**Comment by Steffo99** [comment]
Yes, I still can't see the Quick Sell buttons, on both Chrome and Firefox Nightly.

**Comment by Steffo99** [comment]
After a computer restart, the bug is fixed in Chrome.

**Comment by Steffo99** [comment]
I don't know if it's intended or not, but the buttons only appear in the Steam inventory tab and not in the others.

**Comment by JoiNNN** [comment]
@Steffo99 it is indeed intended.
@Ryahn is this fixed for you too with the latest release?

**Comment by VanityFox** [comment]
So, Enhanced Steam is discontinued. There is now [Augmented Steam](https://github.com/tfedor/AugmentedSteam/). Perhaps your requested feature/bugfix is already implemented there! If not, you could move your issue there.
